ros2pkg/ros2pkg/api/create.py

```diff
@@ -18,7 +18,10 @@
 
 import em
 
-RESOURCE_PATH = os.path.join(os.path.dirname(__file__), '..', 'resource')
+from ros2pkg.api import get_prefix_path
+
+RESOURCE_PATH = os.path.abspath(
+    os.path.join(get_prefix_path('ros2pkg'), 'share', 'ros2pkg', 'resource'))
```

Comment:
Looking at this again I don't think we should couple the resource lookup with the ament index. Instead use the  `pkg_resources` API for this: like https://github.com/ament/ament_package/blob/28308125089b607f2797abee577371d7ee9a0934/ament_package/templates.py#L24

Reply:
so install templates with `package_data` instead of `data_files`, is that right? I think that's the answer to the "how do I install to site-packges" question I was having, good to know that this exists (and that we don't have to specify each file manually!)